admin-dev/themes/new-theme/js/pages/currency/form/components/CurrencyFormatForm.vue

```diff
@@ -27,6 +27,7 @@
     <div class="col-4">
       <h4>{{ $t('step.symbol') }}</h4>
       <input
+        id="custom-symbol"
```

Comment:
If it's not use anywhere but tests, you should use `data-role` instead of `id`.

Reply:
Fixed